ipet/TestRun.py

```diff
@@ -379,13 +380,13 @@ def emptyCurrentProblemData(self):
         """
         return self.currentproblemdata == {}
 
-    def printToConsole(self, formatstr = "{idx}: {d}"):
+    def printToConsole(self, formatstr = "{idx}: {d}", all_rows = False):
         """ Print data to console
         """
+        if (all_rows):
+            pd.set_option('display.max_rows', None)
```

Comment:
I find this change a bit too intrusive. Also, I don't see the benefit: I thought that the two removed lines further down take care of displaying all rows. The change you suggest could be executed in ipet-parse directly, with no need to introduce this argument down through all involved methods.